packages/wdio-allure-reporter/tests/utils.test.ts

```diff
@@ -41,59 +41,49 @@ describe('utils', () => {
     })
 
     describe('getTestStatus', () => {
-        it('return  status for jasmine', () => {
```

Comment:
But this logic is only return FAILED. Could you explain me why we are doing this here? right now, jasmine and mocha return a similar structure, why we cannot consider the error fields instead?

Reply:
@lararojasmr  It seems I did not explain well enough
> why we cannot consider the error fields instead?

We can, and this is a good substitute of config check solution. Please add test to branch
```
} else if (test.errors) {
        return AllureStatus.FAILED
    }
```

Reply:
>right now, jasmine and mocha return a similar structure, why we cannot consider the error fields instead?

Jasmine has soft assertions that can return multiple errors and we can't properly calculate the status, so we always set it as failed.

Reply:
@BorisOsipov What is the logic for jasmine that you are seeing? I removed that logic (from getTestStatus) because after testing both frameworks i saw that the behavior is similar and it does not need specific implementation for the actions that the allure service do.

Reply:
>What is the logic for jasmine that you are seeing? I removed that logic (from getTestStatus) because after testing both frameworks i saw that the behavior is similar and it does not need specific implementation for the actions that the allure service do.

It depends which asserts you use, jasmine or wdio-expect. I'd prefer not to change the logic, at least for this PR. I I don't want to test it now with all the combimation due, I'm not a Jasmine user :)

`test.errors` value sets only for Jasmine integration. It always empty for mocha\cucumber. And I want to see that when we use Jasmine we always have FAILED test status. And I want to see test that when `test.errors` is set to status always failed to maintain test coverage.

Reply:
It is just a matter of adding a unit test for this change.